pkg/cache/context_test.go

```diff
@@ -50,15 +50,53 @@ import (
 )
 
 const (
-	Host1  = "HOST1"
-	appID1 = "app00001"
-	appID2 = "app00002"
-	appID3 = "app00003"
-
-	pod1UID      = "task00001"
-	taskUID1     = "task00001"
-	pod1Name     = "my-pod-1"
+	Host1 = "host0001"
+	Host2 = "host0002"
+
+	appID1             = "app00001"
+	appID2             = "app00002"
+	appID3             = "app00003"
+	appID4             = "app00004"
+	appID5             = "app00005"
+	non_existing_appID = "app-none-exist"
+
+	uid1  = "uid_0001"
+	uid2  = "uid_0002"
+	uid3  = "uid_0003"
+	uid4  = "uid_0004"
+	uid5  = "uid_0005"
+	uid0n = "uid_0000"
+
+	pod1UID     = "task00001"
+	pod2UID     = "task00002"
+	taskUID1    = "task00001"
+	taskUID2    = "task00002"
+	taskUID3    = "task00003"
+	taskUID4    = "task00004"
+	taskUnknown = "non_existing_taskID"
+
+	podName1     = "pod1"
+	podName2     = "pod2"
+	podName3     = "pod3"
+	podName4     = "pod4"
+	podNamespace = "yk"
+
+	nodeName1    = "node1"
+	nodeName2    = "node2"
 	fakeNodeName = "fake-node"
+
+	queueNameA       = "root.a"
+	queueNameB       = "root.b"
+	queueNameC       = "root.c"
+	queueNameUnknown = "root.other"
+
+	// appID1 = "yunikorn-test-00001"
+	// appID2 = "yunikorn-test-00002"
+	// appID3 = "yunikorn-test-00003"
+	// appID4 = "yunikorn-test-00004"
+	// appID5 = "yunikorn-test-00005"
```

Comment:
The code could be removed.

Reply:
Yeah. Forgot to delete them. Update in the new commit. Thank you so much.